modules/olm-filtering-fbc.adoc

```diff
@@ -11,9 +11,9 @@ endif::[]
 
 :_content-type: PROCEDURE
 [id="olm-filtering-fbc_{context}"]
-= Filtering a file-based catalog image
+= Updating or filtering a file-based catalog image
 
-You can use the `opm` CLI to filter, or prune, a catalog image that uses the file-based catalog format. By extracting and updating the contents of an existing catalog image, you can remove one or more Operator packages from the catalog, and then rebuild the image as an updated version.
+You can use the `opm` CLI to update or filter, also known as prune, a catalog image that uses the file-based catalog format. By extracting and modifying the contents of an existing catalog image, you can update, add, or remove one or more Operator package entries from the catalog. You can then rebuild the image as an updated version of the catalog.
```

Comment:
nit:  I find that the subclause added inline distracts/complicates the main clause.  I know you're trying to keep in 'prune' for SEO, but I'd favor either
```suggestion
You can use the `opm` CLI to update or filter --also known as prune-- a catalog image that uses the file-based catalog format. By extracting and modifying the contents of an existing catalog image, you can update, add, or remove one or more Operator package entries from the catalog. You can then rebuild the image as an updated version of the catalog.
```
or use parentheses in whole 
```
...update or filter (also known as prune) ...
```
or in part 

```
...update or filter (prune)...
```

Reply:
Went with parentheses in whole.